commerce/domain/shipping/splitshipment/SplitShipmentUnitTest.cls

```diff
@@ -0,0 +1,244 @@
+/**
+ * @description Sample unit test for SplitShipmentSample and SplitShipmentCallsSuper.
+ */
+@IsTest
+global class SplitShipmentUnitTest {
+    private static final String CART_NAME = 'My Cart';
+    private static final String ACCOUNT_NAME = 'My Account';
+    private static final String WEBSTORE_NAME = 'My WebStore';
+    private static final String DELIVERYGROUP_NAME = 'Default Delivery Group';
+    private static final String CART_ITEM1_NAME = 'My Cart Item 1';
+    private static final String CART_ITEM2_NAME = 'My Cart Item 2';
+    private static final String CART_ITEM3_NAME = 'My Cart Item 3';
+    private static final String SKU1_NAME = 'My SKU 1';
+    private static final String SKU2_NAME = 'My SKU 2';
+    private static final String SKU3_NAME = 'My SKU 3';
+
+    private static final MockDefaultSplitShipmentService mockService = new MockDefaultSplitShipmentService();
+    private static final SplitShipmentCallsSuper splitShipmentCallsSuper = new SplitShipmentCallsSuper(mockService);
+    private static final SplitShipmentSample splitShipmentSample = new SplitShipmentSample();
+
+    /**
+     * @description Verify that SplitShipmentService invoked the default implementation.
+     */
+    @IsTest
+    public static void splitShipmentServiceShouldCallDefaultImplementation() {
+        // Arrange
+        // Create a Cart
+        CartExtension.Cart cart = CartExtension.CartTestUtil.createCart();
+
+        // Create item arrangement request
+        CartExtension.ItemArrange itemArrange = createItemArrange(cart.getCartItems().get(0).getId(), null, null);
+        List<CartExtension.ItemArrange> itemArrangeList = new List<CartExtension.ItemArrange>();
+        itemArrangeList.add(itemArrange);
+
+        CartExtension.ItemArrangementRequest request = new CartExtension.ItemArrangementRequest.Builder()
+                .withCart(cart)
+                .withItemArrangeList(itemArrangeList)
+                .build();
+
+        // Act
+        Test.startTest();
+        splitShipmentCallsSuper.arrangeItems(request);
+        Test.stopTest();
+
+        // Assert
+        // Verify that the default implementation was called
+        Assert.isTrue(cart.getName().contains('Default Split Shipment Service was invoked'));
+    }
+
+    /**
+     * @description Verify that SplitShipmentService arranged CartItems by a "Product Conveyability" custom field.
+     * The 'IsConveyable__c' custom checkbox field on Product2 signifies if the product can go on the conveyer belt
+     * at a warehouse. Items that cannot go on the conveyer belt may be big, bulky, and/or heavy which may require
+     * additional packaging and will take longer to get onto a delivery truck.
+     */
+    @IsTest
+    public static void splitShipmentServiceShouldCreateDeliveryGroupBasedOnConveyability() {
```

Comment:
Do we need another test case that we don't create delivery group #3 when non-coveyable dg has been created?

Reply:
Another test case isn't necessary, but I'll add another non-conveyable product in this test to cover our bases.

Reply:
@pplat23 this test case already verifies this. We've added 3 items to cart, 2 of are non-conveyable products [here](https://github.com/forcedotcom/commerce-extensibility/pull/44/files#diff-0103db420f68fbe11d86b7ee488593e8dd5d4a0d05faa615e09ba5a8b334855bR161-R190) and have verified that only 2 Cart Delivery Groups with the correct items were created [here](https://github.com/forcedotcom/commerce-extensibility/pull/44/files#diff-0103db420f68fbe11d86b7ee488593e8dd5d4a0d05faa615e09ba5a8b334855bR90-R108).